File: SEM2/Atelier/TP2/ex3.c
#include <stdio.h> 


int main () {
    int n ; 
    printf("Enter The length  \n") ; 
    scanf("%d", &n) ;
    int arr[n] ;
    int *p = arr ; 
    while(p < arr+n)
    {
        printf("Enter The element \n") ; 
        scanf("%d", p) ;  
        ++p ; 
    }
    
    
    int sum = 0 ; 
    int *p1 = arr;
    while (p1 < arr + n) {
        sum += *p1 ; 
        p1++ ; 
    }

    printf("The Sum is : %d" , sum) ; 
    

    return 0 ; 
}

File: SEM2/PYTHON/TP4/ex3.py
arr = []
while True : 
    d = int(input("Enter Data Integer "))
    arr.append(d)
    l = input("Quit ? y-> quit , else stay ")
    if l.upper() == "Y" :
        break 
sumF = 0 
for data in arr :
    if data % 5 == 0 :
        sumF +=1 
print(f"total numbers that can be devided by 5 : {sumF}") 
evenSum = 0 
for data in arr :
    if data % 2 == 0 :
        evenSum +=1 
print(f"total even numbers : {evenSum}")
sum = 0
for data in arr :
    sum += data 
sum /= len(arr) 
check = sum >= 10 
